src/platform/forms-system/src/js/fields/AutosuggestField.jsx

```diff
@@ -208,6 +208,19 @@ export default class AutosuggestField extends React.Component {
     const { idSchema, formContext, formData, uiSchema, schema } = this.props;
     const id = idSchema.$id;
 
+    // wrap matching text in a <mark> element
+    const highlightText = uiSchema['ui:options']?.highlightText ?? true;
+    const value = this.state.input?.toLowerCase() || '';
+    const regexp = new RegExp(`(${value})`, 'i');
+    const highLightMatchingText = query => {
+      if (value.length > 2) {
+        return query
+          .split(regexp)
+          .map(str => (str.toLowerCase() === value ? <mark>{str}</mark> : str));
```

Comment:
Same question for @noahgelman as before: Does this pose a problem for screen readers?